ci/parse-trivy-results.py

```diff
@@ -0,0 +1,124 @@
+
+1) Reads in results from two files (json and table format) generated by Trivy.
+2) Creates a GitHub issue in the given repo if any vulnerabilities were found.
+3) Fails the CI build if any vulerabilities of HIGH/CRITICAL severity exist.
+
+Both the json and table Trivy results must be created prior to running
+this script. These can be generated with:
+    trivy image --format json --light --no-progress -o <trivy_output>.json
+        gcr.io/cre-tools/ts-bridge
+    trivy image --light  --no-progress -o <trivy_output>.table
+        gcr.io/cre-tools/ts-bridge
+
+Usage:
+    python3 parse-trivy-results.py <trivy_output> <commit_id> <build_id> <release_tag> <repo_name> <token_file>
+
+Example usage:
+    python3 parse-trivy-results.py trivy-out 99637b2 c933bec7-9acd-48e2-bd7f-7ed7251b5792 1.1.1 google/ts-bridge git_token.txt
+
+"""
+import json
+import sys
+from github import Github
+
+CMDLINE_ARGS = 7
+
+def load_results():
+    """ Load the results from Trivy."""
+    trivy_out_json = "{}.json".format(sys.argv[1])
+    trivy_out_table = "{}.table".format(sys.argv[1])
+    print(trivy_out_table, trivy_out_json)
+    try:
+        with open(trivy_out_json) as f:
+            # Index is 0 because there is only one target built.
+            trivy_result = json.load(f)[0]
+        with open(trivy_out_table) as f:
+            trivy_table = f.read()
+    except FileNotFoundError:
+        error = ("Please run \n"
+                "trivy image --format json --light --no-progress -o {} "
+                "gcr.io/cre-tools/ts-bridge \n"
+                "trivy image --light  --no-progress -o {} "
+                "gcr.io/cre-tools/ts-bridge \n"
+                ).format(trivy_out_json, trivy_out_table)
+        sys.exit(error)
+    return [trivy_result, trivy_table]
+
+def get_severity_list(vulnerabilities):
+    """Filters out all the severities in a given list of vulnerabilities."""
+    severity_list = [v.get('Severity') for v in vulnerabilities]
+    return sorted(set(severity_list))
+
+def get_github_repo():
+    """Connects to GitHub API and returns a repo object."""
+    repo_name = sys.argv[5]
+    with open(sys.argv[6]) as f:
+        token = f.read()
+    github = Github(token)
+    return github.get_repo(repo_name)
+
+def create_issue(target_name, num_vulnerabilities, severity_list, table):
+    """Creates a Github issue with vulnerabilities as description."""
+    commit_id = sys.argv[2]
+    build_id = sys.argv[3]
+    release_tag = sys.argv[4]
+    repo = get_github_repo()
+
+    title = ("Vulnerability [{}] found in {}: Images from commit {} cannot be "
+             "released").format(",".join(severity_list), release_tag, commit_id)
+
+    intro = ("Trivy has detected {} vulnerabilities in your latest "
+             "build. Please correct this issue so the new images can be "
+             "published on Container Registry.\n").format(num_vulnerabilities)
+    body = [intro]
+    body.append("**Cloud Build ID:** {}".format(build_id))
```

Comment:
Does the long brace format work here? E.g.:

```
body.append ("**Commit ID:** {}".format(commit_id)
                        "**Tag:** {}".format(release_tag)
                        "**Target:** {}".format(target_name))
```         

Not entirely sure how pythonic this is so you can ask the readability reviewer too.